$OpenBSD: patch-configure,v 1.4 2010/09/30 07:50:14 jasper Exp $

Fix some stupid egrep(1) misusage causing this function to always
return 1.

--- configure.orig	Tue Sep 28 20:25:59 2010
+++ configure	Tue Sep 28 21:07:53 2010
@@ -13683,7 +13683,7 @@ disabled_plugins=
 # Plugin specific checks
 # ================================================================
 check_plugin_defined() {
-	if echo "$2" | egrep "\b($1)\b" > /dev/null;
+	if echo "$2" | egrep "$1" > /dev/null;
 	then
 		return 1
 	else
